probe/src/probe/publisher/publisher.cpp

```diff
@@ -245,9 +227,8 @@ px::Status publisher::consume_uprobe_data(uint64_t table_id, px::types::TabletID
         auto tinfo = m_inspector->get_thread_ref(pid, true, true, true);
         if (tinfo) {
             gevt.container_id = tinfo->m_container_id;
-            // std::cout << "[qianlu] find container_id for pid:" << pid << " container_id:" << gevt.container_id << std::endl;
         } else {
-            std::cout << "[qianlu] cannot find container_id for pid:" << pid << std::endl;
+		LOG(WARNING) << "[stirling] cannot find container_id for pid:" << pid << std::endl;
```

Comment:
I think there will be performance problems here

Reply:
You mean this block will be executed frequently?

Reply:
How frequently would it log?

Reply:
It logs when we failed to find the corresponding `container_id` from `sinsp`.

Reply:
I think it's normal for the virtual machine process to have a request. Is it necessary to print this line of log